syntax = "proto3";

message RequestMsg
{
    int32 cmdType = 1;  // 标记 1：密钥协商 2:密钥校验 3:密钥注销
    bytes clientID = 2; // 客户端ID
    bytes serverID = 3; // 服务器ID
    bytes sign = 4;     // data的签名
    bytes data = 5;     // 业务数据
}

message RespondMsg
{
    bool status = 1;    // 客户端请求的状态
    int32 seckeyID = 2; 
    bytes clientID = 3; // 客户端ID
    bytes serverID = 4; // 服务器ID
    bytes data = 5;     // 业务数据
}
